packages/api/src/utils/server/genericJsonServer.ts

```diff
@@ -42,7 +42,7 @@ export function getGenericJsonServer<
 
       handler: async function handler(this: FastifyInstance, req, resp): Promise<unknown | void> {
         const args: any[] = routeSerdes.parseReq(req as ReqGeneric as ReqTypes[keyof Api]);
-        const data = (await api[routeId](...args)) as Resolves<Api[keyof Api]>;
+        const data = (await api[routeId](...args, req, resp)) as Resolves<Api[keyof Api]>;
```

Comment:
Types are already correct but request / response objects were never passed in here.

Reply:
Types are updated and no longer support passing in request / response objects to API implementation